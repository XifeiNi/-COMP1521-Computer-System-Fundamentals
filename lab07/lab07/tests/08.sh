./vmsim 8 2 < tests/trace04
