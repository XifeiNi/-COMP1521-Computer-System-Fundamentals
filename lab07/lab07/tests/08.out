
@ t=0, read from pA=0 (vA=0)

PageTable (Stat,Acc,Frame)
[ 0]  L,  0,  0 @ 0
[ 1]  -, -1, -1
[ 2]  -, -1, -1
[ 3]  -, -1, -1
[ 4]  -, -1, -1
[ 5]  -, -1, -1
[ 6]  -, -1, -1
[ 7]  -, -1, -1
MemFrames
[ 0]  0 @ 0
[ 1] -1 @ 4096

@ t=1, read from pA=4096 (vA=4096)

PageTable (Stat,Acc,Frame)
[ 0]  L,  0,  0 @ 0
[ 1]  L,  1,  1 @ 4096
[ 2]  -, -1, -1
[ 3]  -, -1, -1
[ 4]  -, -1, -1
[ 5]  -, -1, -1
[ 6]  -, -1, -1
[ 7]  -, -1, -1
MemFrames
[ 0]  0 @ 0
[ 1]  1 @ 4096

@ t=2, read from pA=0 (vA=8192)

PageTable (Stat,Acc,Frame)
[ 0]  -, -1, -1
[ 1]  L,  1,  1 @ 4096
[ 2]  L,  2,  0 @ 0
[ 3]  -, -1, -1
[ 4]  -, -1, -1
[ 5]  -, -1, -1
[ 6]  -, -1, -1
[ 7]  -, -1, -1
MemFrames
[ 0]  2 @ 0
[ 1]  1 @ 4096

@ t=3, read from pA=4096 (vA=12288)

PageTable (Stat,Acc,Frame)
[ 0]  -, -1, -1
[ 1]  -, -1, -1
[ 2]  L,  2,  0 @ 0
[ 3]  L,  3,  1 @ 4096
[ 4]  -, -1, -1
[ 5]  -, -1, -1
[ 6]  -, -1, -1
[ 7]  -, -1, -1
MemFrames
[ 0]  2 @ 0
[ 1]  3 @ 4096

@ t=4, read from pA=0 (vA=16384)

PageTable (Stat,Acc,Frame)
[ 0]  -, -1, -1
[ 1]  -, -1, -1
[ 2]  -, -1, -1
[ 3]  L,  3,  1 @ 4096
[ 4]  L,  4,  0 @ 0
[ 5]  -, -1, -1
[ 6]  -, -1, -1
[ 7]  -, -1, -1
MemFrames
[ 0]  4 @ 0
[ 1]  3 @ 4096

@ t=5, read from pA=4096 (vA=20480)

PageTable (Stat,Acc,Frame)
[ 0]  -, -1, -1
[ 1]  -, -1, -1
[ 2]  -, -1, -1
[ 3]  -, -1, -1
[ 4]  L,  4,  0 @ 0
[ 5]  L,  5,  1 @ 4096
[ 6]  -, -1, -1
[ 7]  -, -1, -1
MemFrames
[ 0]  4 @ 0
[ 1]  5 @ 4096

@ t=6, read from pA=0 (vA=24576)

PageTable (Stat,Acc,Frame)
[ 0]  -, -1, -1
[ 1]  -, -1, -1
[ 2]  -, -1, -1
[ 3]  -, -1, -1
[ 4]  -, -1, -1
[ 5]  L,  5,  1 @ 4096
[ 6]  L,  6,  0 @ 0
[ 7]  -, -1, -1
MemFrames
[ 0]  6 @ 0
[ 1]  5 @ 4096

@ t=7, read from pA=4096 (vA=28672)

PageTable (Stat,Acc,Frame)
[ 0]  -, -1, -1
[ 1]  -, -1, -1
[ 2]  -, -1, -1
[ 3]  -, -1, -1
[ 4]  -, -1, -1
[ 5]  -, -1, -1
[ 6]  L,  6,  0 @ 0
[ 7]  L,  7,  1 @ 4096
MemFrames
[ 0]  6 @ 0
[ 1]  7 @ 4096

@ t=8, read from pA=0 (vA=0)

PageTable (Stat,Acc,Frame)
[ 0]  L,  8,  0 @ 0
[ 1]  -, -1, -1
[ 2]  -, -1, -1
[ 3]  -, -1, -1
[ 4]  -, -1, -1
[ 5]  -, -1, -1
[ 6]  -, -1, -1
[ 7]  L,  7,  1 @ 4096
MemFrames
[ 0]  0 @ 0
[ 1]  7 @ 4096

@ t=9, read from pA=4096 (vA=4096)

PageTable (Stat,Acc,Frame)
[ 0]  L,  8,  0 @ 0
[ 1]  L,  9,  1 @ 4096
[ 2]  -, -1, -1
[ 3]  -, -1, -1
[ 4]  -, -1, -1
[ 5]  -, -1, -1
[ 6]  -, -1, -1
[ 7]  -, -1, -1
MemFrames
[ 0]  0 @ 0
[ 1]  1 @ 4096

@ t=10, read from pA=0 (vA=8192)

PageTable (Stat,Acc,Frame)
[ 0]  -, -1, -1
[ 1]  L,  9,  1 @ 4096
[ 2]  L, 10,  0 @ 0
[ 3]  -, -1, -1
[ 4]  -, -1, -1
[ 5]  -, -1, -1
[ 6]  -, -1, -1
[ 7]  -, -1, -1
MemFrames
[ 0]  2 @ 0
[ 1]  1 @ 4096

@ t=11, read from pA=4096 (vA=12288)

PageTable (Stat,Acc,Frame)
[ 0]  -, -1, -1
[ 1]  -, -1, -1
[ 2]  L, 10,  0 @ 0
[ 3]  L, 11,  1 @ 4096
[ 4]  -, -1, -1
[ 5]  -, -1, -1
[ 6]  -, -1, -1
[ 7]  -, -1, -1
MemFrames
[ 0]  2 @ 0
[ 1]  3 @ 4096

@ t=12, read from pA=0 (vA=16384)

PageTable (Stat,Acc,Frame)
[ 0]  -, -1, -1
[ 1]  -, -1, -1
[ 2]  -, -1, -1
[ 3]  L, 11,  1 @ 4096
[ 4]  L, 12,  0 @ 0
[ 5]  -, -1, -1
[ 6]  -, -1, -1
[ 7]  -, -1, -1
MemFrames
[ 0]  4 @ 0
[ 1]  3 @ 4096

@ t=13, read from pA=4096 (vA=20480)

PageTable (Stat,Acc,Frame)
[ 0]  -, -1, -1
[ 1]  -, -1, -1
[ 2]  -, -1, -1
[ 3]  -, -1, -1
[ 4]  L, 12,  0 @ 0
[ 5]  L, 13,  1 @ 4096
[ 6]  -, -1, -1
[ 7]  -, -1, -1
MemFrames
[ 0]  4 @ 0
[ 1]  5 @ 4096

@ t=14, read from pA=0 (vA=24576)

PageTable (Stat,Acc,Frame)
[ 0]  -, -1, -1
[ 1]  -, -1, -1
[ 2]  -, -1, -1
[ 3]  -, -1, -1
[ 4]  -, -1, -1
[ 5]  L, 13,  1 @ 4096
[ 6]  L, 14,  0 @ 0
[ 7]  -, -1, -1
MemFrames
[ 0]  6 @ 0
[ 1]  5 @ 4096

@ t=15, read from pA=4096 (vA=28672)

PageTable (Stat,Acc,Frame)
[ 0]  -, -1, -1
[ 1]  -, -1, -1
[ 2]  -, -1, -1
[ 3]  -, -1, -1
[ 4]  -, -1, -1
[ 5]  -, -1, -1
[ 6]  L, 14,  0 @ 0
[ 7]  L, 15,  1 @ 4096
MemFrames
[ 0]  6 @ 0
[ 1]  7 @ 4096

#loads = 16, #saves = 0, #replacements = 14
